### ISSUE TITLE
FR: JSDocs should be inherited from an implemented interface/base class

### ISSUE THREAD

**Issue by Jack-Works** [Microsoft/TypeScript#19524]
**TypeScript Version:**  2.6

**Code**

```ts
interface Human {
    /** Name of human */
    name: string
}
class OhHuman implements Human {
    name = 'Jack'
}
(new OhHuman).name
```

**Expected behavior:**
> `(property) OhHuman.name: string` <br />
> Name of human

**Actual behavior:**
> `(Property) OhHuman.name: string`

**Also See:**
https://github.com/Microsoft/TypeScript/issues/12966#issuecomment-339180869

**Comment by andy-ms** [comment]
Fixed by #18804?

**Comment by Jack-Works** [comment]
Yes, I think so. I'll keep track on that pr.
